CondCore/ESSources/test/SiStripTest/noiseanalyzer.cc

```diff
@@ -2,26 +2,29 @@
 #include <string>
 #include <iostream>
 #include <map>
-#include "FWCore/Framework/interface/EDAnalyzer.h"
-#include "FWCore/Framework/interface/Event.h"
-#include "FWCore/Framework/interface/ESHandle.h"
-#include "FWCore/Framework/interface/MakerMacros.h"
-#include "FWCore/Framework/interface/EventSetup.h"
 
 #include "CondFormats/Calibration/interface/mySiStripNoises.h"
 #include "CondFormats/DataRecord/interface/mySiStripNoisesRcd.h"
 
+#include "FWCore/Framework/interface/EDAnalyzer.h"
```

Comment:
why can't this be `one::` or even better `global::`.
Also please `cout` -> `LogPrint`

Reply:
> why can't this be `one::` or even better `global::`. 

Yes I forgot the update this one! I will fix this in the next commit, thanks for spotting this!

> Also please `cout` -> `LogPrint`

Is this policy enforced/preferred also in `test` directory?

Reply:
> Is this policy enforced/preferred also in test directory?

since you are touching it anyway, better safe than sorry.

Reply:
ok! will do in the next push then!

Reply:
all done in ce3e23b